### PR TITLE
fix pa scrapers

### PR DESCRIPTION
**Pull request by sglyon**
- bring in updates from NC pr
- fix pa-scrapers

### CODE REVIEW

**Comment by sglyon** [comment]
all PA scrapers (standard vaccine one and 4 demographic ones) were broken. This fixes them

File: can_tools/scrapers/base.py [hunk 1]
```diff
@@ -274,6 +276,9 @@ def extract_CMU(
         var_name: str
             The name of the column in `df` that should be used to lookup
             items from the `cmu` dict for unpacking columns
+        skip_columns: List[str] (default=[])
+            Can be set instead of ``columns`` if there are a small number
+            of columns that should not be set
```
Comment (mikelehen) [inline can_tools/scrapers/base.py:281]: Can you clarify what the contract is between this and `columns`?  The comment suggests you can't use them together, but the code looks like they're used together (you'd only add a column to `skip_columns` if it was already in `columns`?)...

File: can_tools/scrapers/official/PA/pa_vaccines.py [hunk 2]
```diff
@@ -124,7 +124,7 @@ def normalize(self, resjson):
 
         # Dump records into a DataFrame
         df = pd.DataFrame.from_records(data_rows).dropna()
-        df = df.query("location_name != '' & location_name != 'Out-of-State'")
+        df = df.query("location_name != '' & location_name != 'Out-of-State*'")
```
Comment (mikelehen) [inline can_tools/scrapers/official/PA/pa_vaccines.py:127]: Just to confirm, we don't have to worry about the other location names that the demographics scraper does?  https://github.com/covid-projections/can-scrapers/pull/252/files#diff-110778fd367ad022afb439a15950c6a6b7257e77cf95c7d2d86dcd296956bcd0R194

File: can_tools/scrapers/official/PA/pa_vaccines.py [hunk 3]
```diff
@@ -355,62 +322,37 @@ def normalize(self, resjson):
...
             .pivot_table(
-                index=["location_name", self.demographic],
-                columns="variable",
-                values="value",
+                index=["county", self.demographic],
+                columns="coverage",
+                values="count",
+                aggfunc="sum",
             )
+            # .fillna(0)
```
Comment (mikelehen) [inline can_tools/scrapers/official/PA/pa_vaccines.py:331]: intentionally left here?

**Review by mikelehen — APPROVED** [review]
Unfortunately I don't know the code well enough to do a terribly deep review, but this looks good superficially at least.  😁

File: can_tools/scrapers/base.py (hunk 1 above)
Comment (sglyon) [reply can_tools/scrapers/base.py:281]: Yeah this is a bit hacky.

I use it when I want to preserve the value of one of these 7 columns (category, measurement, unit, age, race, ethnicity, sex), but set the value of the other 6. In this PR, I leave `columns` as the default and then set `skip_column = ["age"]` to make sure the age column is not overwritten in this function.

A bit wonky, I agree

File: can_tools/scrapers/official/PA/pa_vaccines.py (hunk 2 above)
Comment (sglyon) [reply can_tools/scrapers/official/PA/pa_vaccines.py:127]: Correct!

File: can_tools/scrapers/official/PA/pa_vaccines.py (hunk 3 above)
Comment (sglyon) [reply can_tools/scrapers/official/PA/pa_vaccines.py:331]: deleted, thanks

**Pull request merged**